apps/right-to-rent-check/translations/src/en/fields.json

```diff
@@ -67,16 +67,16 @@
     "legend": "Additional details",
     "options": {
       "reference-number": {
-        "label": "<h3>Home Office reference number</h3>If the person has applied for a visa with the Home Office, they will be issued with a reference number. This could be an application or appeal number, an Application Registration Card (ARC) number or a case ID number."
+        "label": "<h2>Home Office reference number</h2>This could be an application or appeal number, an Application Registration Card (ARC) number or a case ID number."
```

Comment:
Why does it _need_ to be an `h2`, out of interest?

Reply:
From what I've learnt the general principle is to have h tags in order.  This is because of the way users use screenreaders, they use shortcut keys to jump between the tags.  It's not the end of the world but is a bit strange and a little annoying for those users from what I understand.   
https://ux.stackexchange.com/questions/49111/is-it-ok-to-use-h1-and-then-h3-instead-of-h1-and-then-h2